### DISCUSSION TITLE
Does it work with latest ROS 2 version

### DISCUSSION THREAD

**Discussion by weimfau** [iRobotEducation/create3_docs#451] [Questions] [answered]
Hi,

I am using ROS 2 Iron since 'galactic' has reached end-of life. Should I expect Create 3 work with Iron. I could not see any ros2 topic by 'ros2 topic list'. Here are my settings.
(1) Create 3 is connected to my laptop via 'usb_c_to_ethernet_adaptor  -- ethernet_cable --- usb_c_to_ethernet_adaptoer'
(2) Both create 3 and my laptop's RMW_IMPLEMENTATION are set to rmw_cyclonedds_cpp
(3) I could access the website http://192.168.186.2

Thank you very much. Any help is greatly appricated.

**Comment by weimfau** [comment]
By the way, I am not using wifi connection.

**Comment by alsora** [comment, marked as answer]
Hi @weimfau, ROS 2 does not guarantee that different distributions (e.g. galactic and iron) can communicate between each others.
As of today, Create 3 firmwares are only available for Galactic and Humble, so you should use one of those two (make sure to install the correct firmware on the robot).

We are currently working on an Iron release and we will let you know once it's available.

**Reply by weimfau** [reply]
Thank you very much for the update.
